dealiiadapter.cc

```diff
@@ -378,14 +383,30 @@ class ElasticProblem
     Vector<double> old_displacement;
     Vector<double> displacement;
     Vector<double> old_forces;
+    Vector<double> forces;
     Vector<double> system_rhs;
+    //precice related initializations
+    int             mesh_id;
+    int             read_data_id;
+    int             write_data_id;
```

Comment:
Similar to the actual data, I would prefer to name these IDs `displacements_id` and `forces_id` or something similar.